### ISSUE TITLE
Improve accessibility of the segmented tabs switcher

### ISSUE THREAD

**Issue by MarcoZehe** [Automattic/wp-calypso#38651]
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/posts or https://wordpress.com/stats, as examples.
2. If the screen is big enough, the element with role 'section-nav-tabs_list' is designated a role of "menu", and the items with class 'section-nav-tab_link' are assigned a role ="menuitem". Navigate to these items using a screen reader and listen to the menu items being read as such, inside a "menu" container.

#### What I expected

On big enough screen sizes, these act more like tabs than a menu. A menu usually is a dropdown or popup of some sort, not something that is permanently visible. Incidentally, the component that gets used even has "tabs" in the name, and the class lists as well. I expected these to be tabs, not a menu.

#### What happened instead

These items are announced as a menu.

#### Browser / OS version

Firefox 71 on Windows 10 with NVDA 2019.2.1
.
#### Screenshot / Video

n/a

#### Context / Source

#accessibility #empathy #manual-testing #user-report

**Comment by worldomonation** [comment]
Hi! I'm performing triage on older bugs as part of Quality Squad's efforts to cut down on backlog issue counts.

There has been no activity on this issue in over a year.
With that said, a11y is something that should be emphasized at Automattic and so this issue should remain open.
Adding the `OSS Citizen` label and will investigate if there exists an a11y project or milestone that this issue can be put on.
